code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm

```diff
@@ -17,24 +17,30 @@
 	layer = GAS_SCRUBBER_LAYER
 	hide = TRUE
 	shift_underlay_only = FALSE
+	pipe_state = "uvent"
+	vent_movement = VENTCRAWL_ALLOWED | VENTCRAWL_CAN_SEE | VENTCRAWL_ENTRANCE_ALLOWED
 
+	///Direction of pumping the gas (RELEASING or SIPHONING)
 	var/pump_direction = RELEASING
-
+	///Flag to indicate what pressure check to implement (EXT_BOUND, INT_BOUND, NO_BOUND)
 	var/pressure_checks = EXT_BOUND
+	///Variable to set the external pressure check (default 101 kPa)
 	var/external_pressure_bound = ONE_ATMOSPHERE
+	///Variable to set the internal pressure check (default 0 kPa)
```

Comment:
Maybe "The external/internal pressure threshold"? the defaults are fine, variable to set is a bit confusing